core/runtime/execute_engine.cpp

```diff
@@ -155,8 +155,10 @@ std::vector<at::Tensor> execute_engine(std::vector<at::Tensor> inputs, c10::intr
           std::make_unique<torch::autograd::profiler::RecordProfile>(compiled_engine->output_profile_path);
     }
 
-    for (size_t o = inputs.size(); o < (compiled_engine->num_io.first + compiled_engine->num_io.second); o++) {
-      uint64_t pyt_idx = compiled_engine->out_binding_map[o];
```

Comment:
In the case of the detectron model in #2053, the output binding map was of the form:
```python
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg264_1 has TensorRT binding index: 0, Torch binding index: 0
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg268_1 has TensorRT binding index: 1, Torch binding index: 1
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: add_121 has TensorRT binding index: 2, Torch binding index: 2
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: clamp_51 has TensorRT binding index: 3, Torch binding index: 3
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg72_1 has TensorRT binding index: 7, Torch binding index: 4
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg73_1 has TensorRT binding index: 8, Torch binding index: 5
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg74_1 has TensorRT binding index: 9, Torch binding index: 6
DEBUG: [Torch-TensorRT - Debug Build] - Input binding name: arg75_1 has TensorRT binding index: 10, Torch binding index: 7
DEBUG: [Torch-TensorRT - Debug Build] - Output binding name: output0 has TensorRT binding index: 4, Torch binding index: 8
DEBUG: [Torch-TensorRT - Debug Build] - Output binding name: output1 has TensorRT binding index: 5, Torch binding index: 9
DEBUG: [Torch-TensorRT - Debug Build] - Output binding name: output2 has TensorRT binding index: 6, Torch binding index: 10
DEBUG: [Torch-TensorRT - Debug Build] - Output binding name: output3 has TensorRT binding index: 11, Torch binding index: 11
```
Some of the output bindings have TRT indices which are smaller than the number of input bindings, causing errors when the `out_binding_map` is accessed with a fixed index like `o`, as above.